keyboards/pimoroni/keybow2040/config.h

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+#define RGB_MATRIX_LED_COUNT 16
```

Comment:
```suggestion
// I2C config
#define I2C_DRIVER I2CD0
#define I2C1_SDA_PIN GP4
#define I2C1_SCL_PIN GP5
#define I2C1_CLOCK_SPEED 100000

// The board has a single IS31FL3731 chip with the AD pin connected to GND.
#define DRIVER_COUNT 1
#define DRIVER_ADDR_1 0b1110100

#define RGB_MATRIX_LED_COUNT 16
```
I2C configuration options can look confusing, because for historical reasons the names of some defines include the `1` index.  In these names (`I2C1_SDA_PIN`, `I2C1_SCL_PIN`, `I2C1_CLOCK_SPEED`) the `1` index does not refer to the I2C peripheral number of the MCU — it's something like “the index of the I2C bus on the keyboard PCB”, except the QMK code does not actually support more than a single I2C bus yet, therefore the only valid value of that index is 1.  The only place where the actual I2C peripheral number is specified is the value of the `I2C_DRIVER` define — `I2CD0` refers to the `I2C0` peripheral as described in the MCU datasheet.

Reply:
Ohhhhh 💡 that makes a lot more sense than my less charitable assumption!